### PR TITLE
Add grafana/experimental package

### PR DESCRIPTION
**Pull request by zoltanbedi**
**What this PR does / why we need it**:

This pr adds the grafan/experimental package to the workspace.

### CODE REVIEW

**Review by andresmgot — COMMENTED** [review]
I may be missing context. Why is this removing cloudwatch components?

**Comment by zoltanbedi** [comment]
@andresmgot We moved the components to https://github.com/grafana/grafana-experimental so we can reuse them.

**Comment by andresmgot** [comment]
got it, let's wait until 8.3 then to we avoid conflicts with the PR that is in the works

**Review by dprokop — APPROVED** [review]

**Comment by torkelo** [comment]
@zoltanbedi can this be merged now?

**Comment by zoltanbedi** [comment]
@torkelo this needs some tweaking. I'm on it.

**Comment by torkelo** [comment]
@zoltanbedi it looks like it's missing a lot of components as well (Space, FlexItem )

**Comment by zoltanbedi** [comment]
We need a new version of grafana/experimental with this pr in place https://github.com/grafana/grafana-experimental/pull/1/files. I've changed the editor in cloudwatch to use the components from experimental but it is missing the AccessoryButton.

**Review by sunker — COMMENTED** [review]
Tested and it works fine. 

I just have one ask, and that is to sort the imports in the cloudwatch plugin components like this: 
1. built in
2. external 
3. internal

I don't know why this is not enforced by es-lint? Should be a rule for this I think

File: public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/Dimensions.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import React, { useMemo, useState } from 'react';
-import { isEqual } from 'lodash';
 import { SelectableValue } from '@grafana/data';
-import { Dimensions as DimensionsType, CloudWatchMetricsQuery } from '../../types';
-import EditorList from '../ui/EditorList';
+import { EditorList } from '@grafana/experimental';
+import { isEqual } from 'lodash';
+import React, { useMemo, useState } from 'react';
```
[the remark this answers is not in the document]
Comment (zoltanbedi) [reply public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/Dimensions.tsx:2]: sure!

**Comment by torkelo** [comment]
Any chance of merging this soon?

**Comment by torkelo** [comment]
Any chance of merging this soon?

**Comment by zoltanbedi** [comment]
Sorry, I think we can merge this now.

**Pull request merged**